### ISSUE TITLE
More customization for sendAnnouncement

### ISSUE THREAD

**Issue by Wazarr94** [haxball/haxball-issues#868]
Ola,

Just tried the sendAnnouncement function, and I'd suggest some small changes to improve this function and explore new possibilities:

1) The possibility to change the color of the column before the text. It is always red, and I wonder if we could also change its color. It would be a rather easy change to do and you can use the border-color style (which has a priority in front of the color of the border style)

2) Using some sort of Markdown instead of using the style parameter. It would remove one parameter (why not), and would enable more possibilities, and I think it could be used efficiently. For instance, Discord uses **text** for bold, __text__ for underline, *text* for italic, and it's rather easy to use and to combine. You could maybe use ^text^ for small, and you'd get small-bold by entering ^**text**^ or **^text^** and *^text^* or ^*text*^  for small-italic. Up to you.

Thanks for paying attention.

**Comment by Wazarr94** [comment]
The first point seems pretty important after more testing, some colors that could be could are "ruined" by the fact the border is red. The Markdown point might be a bit too much, but I cannot stress enough how we need to interact with the border color.

If modifying parameters annoys you, if you could just make the border the same color as the text, it'll be already really good, thanks

**Comment by akelaa** [comment]
I actually saw a room with yellow border announcements. Probably they modified the headless-min.js, or there is a way to change border color where wiki doesn't mentioned.

**Comment by Wazarr94** [comment]
In the meantime, i'll probably change the headless min file, red everytime is ugly and I cannot stand it

**Comment by uzayyli** [comment]
Yes please let us edit that border color.. Then we could set it = background color and make it "invisible" too :)
